### DISCUSSION TITLE
The entry of the inode was not found

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#1519] [Q&A] [answered]
I'm testing JuiceFS with MariaDB 10.5, and i can observe some of these lines in the log:

`<WARNING>: The entry of the inode was not found. inode: 16085 [sql.go:2318]`

What does those mean and does this require special action?

**Comment by davies** [comment, marked as answer]
@solracsf this log was generated by backuping metadata in background, it's caused by concurrent changes of entries, no action is required.
